pytorch_lightning/trainer/trainer.py

```diff
@@ -502,11 +502,19 @@ def validate(
 
         model_provided = model is not None
         model = model or self.lightning_module
+        if model is None:
+            raise MisconfigurationException(
+                '`model` must be provided to `trainer.validate()` when `trainer.lightining_module` is not set'
```

Comment:
This message would suggest that users need to set `trainer.lightning_module` manually, which is not the case.

```suggestion
                '`model` must be provided to `trainer.validate()` when it hasn't been passed in a previous run'
```